### ISSUE TITLE
Update Vet Center directions link with sr-only solution

### ISSUE THREAD

**Issue by mmiddaugh** [department-of-veterans-affairs/va.gov-cms#13837]
## Description

The Directions links should include text for screen readers to specifically indicate which location the link is providing directions for. Screen reader users may navigate pages by only focusable items, which means they only hear "link Directions (Google Maps)" without the surrounding context. 

Ideally all the directions links on the page should visually say "Get directions on Google Maps" as this is more clear and also consistent with what is being used in the Facility Locator.

This would include the directions link for locations under "Other nearby locations" also.

As noted in [Laura's comment](https://app.zenhub.com/workspaces/sitewide-facilities-639f5253e4b702a32376339e/issues/gh/department-of-veterans-affairs/va.gov-cms/12085#issuecomment-1538980674) on Directions links SHOULD include sr-only text to indicate specific location #12085, Vet Center pages have unique links. However, they use an aria-label rather than the sr-only solution which can cause confusion. This approach could create confusion because of the difference between the visual link text and label. 

## Steps to Recreate

1. Go to a Vet Center main page, like Nashville Vet Center](https://www.va.gov/nashville-vet-center/) or Vet Center [Locations page](https://www.va.gov/nashville-vet-center/locations/)
2. Start screen reading device listed in Environment
3. Navigate the page by tabbing through focusable areas
4. Verify that the Directions links are unclear where they provide directions to, without use of sight.

## Acceptance Criteria

- [ ] The directions links should be programmatically unique and provide information as to the location the directions pertain
- [ ] All directions links visually say "Get Directions on Google Maps"
- [ ] Accessibility review 

## Possible Fixes (optional)

Add screen reader only text:
`<a href="#">Get directions on Google Maps <span class="sr-only">to Erie Health Care Medical Center</span></a>`

This is already done on the Facility Locator search results Directions links for reference 

## WCAG or Vendor Guidance (optional)

* [Ensure that links with the same accessible name serve a similar purpose](https://dequeuniversity.com/rules/axe/3.5/identical-links-same-purpose)

## Screenshots

![image.png](https://images.zenhubusercontent.com/5d8a47393ac3440001d1a2ed/719172f1-cde7-40ea-9306-bd39e6b1c0a8)
![image.png](https://images.zenhubusercontent.com/5d8a47393ac3440001d1a2ed/1b215b06-cf22-4d21-aea0-7ecd9e740215)
